calicoctl/commands/resources.go

```diff
@@ -221,15 +208,18 @@ func executeConfigCommand(args map[string]interface{}, action action) commandRes
 	} else {
 		// We extracted a single resource type with identifiers from the CLI, convert to
 		// a list for simpler handling.
-		resources = []unversioned.Resource{r}
+		resources = []runtime.Object{r}
 	}
 
 	if len(resources) == 0 {
 		return commandResults{err: errors.New("no resources specified")}
 	}
 
 	if log.GetLevel() >= log.DebugLevel {
-		log.Debugf("Resources: %v", resources)
+		for _, v := range resources {
```

Comment:
Resources -> Resource

Reply:
I'm guessing you're talking about the log line below this, and not this one itself?

Reply:
yup